.github/workflows/cypress.yml

```
@@ -0,0 +1,134 @@
+      - name: Apply Database Migrations
+        run: |
+          echo "=== Applying Migrations (Synchronous) ==="
+          docker exec talawa-api-1 /bin/bash -c 'pnpm apply_drizzle_migrations'
+          echo "=== Waiting for Database Stabilization ==="
+          sleep 10
+
+      - name: Verify .env File
+        run: |
+          echo "=== Checking Environment Variables ==="
+          docker exec talawa-api-1 /bin/bash -c 'echo "API_ADMINISTRATOR_USER_EMAIL_ADDRESS: $API_ADMINISTRATOR_USER_EMAIL_ADDRESS"'
+          docker exec talawa-api-1 /bin/bash -c 'echo "API_ADMINISTRATOR_PASSWORD: $API_ADMINISTRATOR_PASSWORD"' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)